docker-compose.dev.yml

```diff
@@ -54,17 +62,25 @@ services:
     command: /bin/bash -c 'python -m venv .worker_env && source .worker_env/bin/activate && pip install -U pip setuptools wheel && pip install -r requirements.txt && ./celery_worker.sh'
     working_dir: /mnt
     volumes:
-      - "./backend:/mnt"
+      - "./backend:/mnt:z"
     user: ${UID:-1000}
     depends_on:
-      - backend
-      - postgres
-      - redis
+      postgres:
+        condition: service_healthy
+      redis:
+        condition: service_started
+      backend:
+        condition: service_healthy
   postgres:
     environment:
       POSTGRES_USER: ibutsu
       POSTGRES_PASSWORD: ibutsu
       POSTGRES_DB: ibutsu
-    image: postgres:14
+    image: postgres:latest
```

Comment:
You don't want to do this. PostgreSQL versions are not compatible with each other, you need to do a manual data migration. If someone is using postgres 14, and then they suddenly get postgres 15, it won't start because it'll detect that the data is in a previous format.

You are welcome to use postgres:15 instead of 14, but don't change to "latest" because that'll cause issues.

Reply:
Incidentally, I learned this the hard way :-(

Reply:
@rsnyman didn't think about it since it's local dev setup only. Good catch, fixed